### PR TITLE
#17 Feature/heroku

### PR DESCRIPTION
**Pull request by vergissberlin**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const app = require('./src/app')
-const port = process.env.NODE_PORT || 3000
-const host = process.env.NODE_HOST || '0.0.0.0'
+const port = process.env.PORT || 3000
+const host = process.env.HOST || '0.0.0.0'
```
Comment (radmen) [inline server.js:3]: Actually, I'd like to keep support for the `NODE_PORT` and `NODE_HOST`. On some systems, setting `PORT`, or `HOST` env variable might cause some issues :sweat_smile: .

Could you add them as fallback values?

**Comment by vergissberlin** [comment]
Hi @radmen,

I've added aliases for the env variables.

Kind regards
André

File: server.js (hunk 1 above)
Comment (vergissberlin) [reply server.js]: I've added aliases for the env variables.

File: app.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "MJML API",
+  "description": "Responsive email as a service. This project exposes mjml as an API service. It's meant* to be compatible with MJML API .",
+  "repository": "https://github.com/vergissberlin/mjml-api",
```
Comment (radmen) [inline app.json:4]:
```suggestion
  "repository": "https://github.com/baethon/mjml-api",
```

I don't want to sound like ass, but I think we should point to the origin repo :)

**Review by radmen — CHANGES_REQUESTED** [review]

**Pull request merged**